dto/src/main/java/org/jboss/pnc/dto/response/BuildConfigCreationResponse.java

```diff
@@ -25,15 +25,26 @@
 import lombok.Data;
 
 /**
+ * The result of Create&Sync call for creating build config with scm url. If the SCM repository config can be created
+ * immediately, new build config will be also created immediately and returned by {@link getBuildConfig} property. If
+ * the repository needs to be synchronized first, {@link getTaskId} property provides id of the synchronization task.
```

Comment:
The link should be `{@link #getTaskId()}`, otherwise it doesn't work, at least in Eclipse.